aws/resource_aws_appautoscaling_scheduled_action.go

```diff
@@ -0,0 +1,222 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/applicationautoscaling"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+const awsAppautoscalingScheduleTimeLayout = "2006-01-02T15:04:05Z"
+
+func resourceAwsAppautoscalingScheduledAction() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsAppautoscalingScheduledActionCreate,
+		Read:   resourceAwsAppautoscalingScheduledActionRead,
+		Delete: resourceAwsAppautoscalingScheduledActionDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"service_namespace": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					value := v.(string)
+					validServiceNamespaces := []string{"ecs", "elasticmapreduce", "ec2", "appstream", "dynamodb"}
```

Comment:
As below - I'm afraid the list of valid service namespaces may/will grow and I'd like us to avoid the maintenance burden associated with it, if possible.

Reply:
Ok, I removed